index.md

```diff
@@ -1,53 +1,52 @@
 # Aries RFCs by Status
```

Comment:
Updating index.md should be left out of the PR.  That is done as a follow up, as it will cause conflicts.